models.go

```diff
@@ -110,6 +127,24 @@ type EditsResponse struct {
 	Usage   EditsResponseUsage    `json:"usage"`
 }
 
+// The inner result of a create embeddings request, containing the embeddings for a single input.
+type EmbeddingsResult struct {
+	// The type of object returned (e.g., "list", "object")
+	Object string `json:"object"`
+	// The embedding data for the input
+	Embedding []float64 `json:"embedding"`
+	Index     int       `json:"index"`
+}
+
+// EmbeddingsResponse is the response from a create embeddings request.
+//
+// See: https://beta.openai.com/docs/api-reference/embeddings/create
+type EmbeddingsResponse struct {
+	Object string             `json:"object"`
+	Data   []EmbeddingsResult `json:"data"`
+	Model  string             `json:"model"`
```

Comment:
Just to double check, is this field included in the response? I don't see this in the docs here: https://beta.openai.com/docs/api-reference/embeddings/create

Reply:
No! Great catch. Removed `Model` and added `Usage`. I updated the test cases as well.